src/app/contextBroker/contextBroker.cpp

```diff
@@ -276,6 +277,7 @@ bool            logDeprecate;
 #define NGSIV1_AUTOCAST_DESC   "automatic cast for number, booleans and dates in NGSIv1 update/create attribute operations"
 #define MQTT_MAX_AGE_DESC      "max time (in minutes) that an unused MQTT connection is kept, default: 60"
 #define LOG_DEPRECATE_DESC     "log deprecation usages as warnings"
+#define MAX_THRESHOLD_DESC     "maximum threshold for notification queue, default: 80%"
```

Comment:
```suggestion
#define MAX_THRESHOLD_DESC     "maximum threshold for notification queue, as a percentage of the maximum queue size, default 80 (for 80% percentage)"
```

Reply:
I understand 0 is the way of disabling queue alarms and should be the default, in order to keep backward compatibility.

Reply:
Updated in https://github.com/telefonicaid/fiware-orion/pull/4375/commits/1ebccb65531531a8dc0856d8de499b6a34d5a733